docs/pages/reference/cli.mdx

```diff
@@ -1329,6 +1329,7 @@ Environment variables configure your tsh client and can help you avoid using fla
 | TELEPORT_CLUSTER | Name of a Teleport root or leaf cluster | cluster.example.com |
 | TELEPORT_LOGIN | Login name to be used by default on the remote host | root |
 | TELEPORT_LOGIN_BIND_ADDR | Address in the form of host:port to bind to for login command webhook | host:port |
+| TELEPORT_LOGIN_BROWSER | Browser to use for SSO Login. Set to `none` to suppress browser opening. | none |
```

Comment:
What's the default value? What are the permissible values?

Reply:
`none` or anything else.  If anything besides `none` is set it will open the system default browser.  I updated the definition as it does not yet support specifying the specific browser to open.

Reply:
Thanks! I think we can make this a lot clearer. If it's set to `none`, does any browser open at all during an SSO login? I would also incorporate this sentence into the description of the variable:

> If anything besides none is set it will open the system default browser.

Also "SSO Login" is not a proper noun.